tests/integration/kubernetes/k8s-guest-pull-image.bats

```diff
@@ -58,6 +59,41 @@ setup() {
     assert_rootfs_count "$node" "$sandbox_id" "1"
 }
 
+@test "Test we can pull a large image inside the guest" {
+    [[ " ${SUPPORTED_NON_TEE_HYPERVISORS} " =~ " ${KATA_HYPERVISOR} " ]] && skip "Test not supported for ${KATA_HYPERVISOR}."
+    kata_pod_with_nydus_config="$(new_pod_config "$unencrypted_image_1" "kata-${KATA_HYPERVISOR}")"
```

Comment:
Should this be `large_image` rather than `unencrypted_image_1` here? If you set it to the smaller one due to the memory overhead being too high at the moment, as discussed I think that having the large image here, but this tested manually and skipped in this automation I think would be useful for us to remember to fix this later.

Reply:
@stevenhorsman  thank you for bringing this up. I agree to skip in this automation. I'll make a note to ensure we revisit this for a potential fix in the future.